packages/dataviews/src/dataviews-layouts/table/style.scss

```diff
@@ -20,6 +20,25 @@
 
 		&.dataviews-view-table__actions-column {
 			text-align: right;
+
+			&.dataviews-view-table__actions-column--sticky {
```

Comment:
Do these need to be nested?

Reply:
I don’t have a strong opinion on this. I placed them inside `.dataviews-view-table__actions-column` because they act as modifiers for it.

Reply:
I would lean towards avoiding nests if possible.

Reply:
Updated!